policyengine_us/variables/gov/states/vt/tax/income/vt_eitc.py

```diff
@@ -12,5 +12,30 @@ class vt_eitc(Variable):
 
     def formula(tax_unit, period, parameters):
         federal_eitc = tax_unit("eitc", period)
-        rate = parameters(period).gov.states.vt.tax.income.credits.eitc.match
+        p = parameters(period).gov.states.vt.tax.income.credits.eitc
+
+        # S.51 (2025): Enhanced EITC for workers without children
+        enhanced_structure_applies = p.enhanced_structure.in_effect
+
+        # Check if tax unit has qualifying children for EITC
+        person = tax_unit.members
+        is_child_dependent = person("is_child_dependent", period)
+        has_qualifying_children = tax_unit.any(is_child_dependent)
+
+        # Different match rates for workers with and without children (2025+)
+        enhanced_rate = where(
+            has_qualifying_children,
+            p.match,  # Match rate for workers with children
+            1.0,  # Match rate for workers without children
+        )
+
```

Comment:
I should have noticed this earlier but we can just make this a single_amount parameter? With the p.match rate with children 1 and above and 100% for children below 1. This allows us to avoid hard coding and we can use `tax_unit_child_dependents` as the threshold